Userland/Utilities/pkg/AvailablePort.cpp

```diff
@@ -52,20 +46,48 @@ ErrorOr<int> AvailablePort::query_details_for_package(HashMap<String, AvailableP
 
     auto& available_port = possible_available_port.release_value();
 
-    outln("{}: {}, {}", available_port.name(), available_port.version(), available_port.website());
+    outln("{}: {}, {}", available_port.name(), available_port.version_string(), available_port.website());
     if (verbose) {
         out("Installed: ");
-        if (is_installed(installed_ports, package_name))
+        auto installed_port = installed_ports.find(package_name);
+        if (installed_port != installed_ports.end()) {
             outln("Yes");
-        else
+
+            out("Update Status: ");
+
+            auto error_or_ap_version = available_port.version_semver();
+            auto error_or_ip_version = installed_port->value.version_semver();
+
+            if (error_or_ap_version.is_error() || error_or_ip_version.is_error()) {
+                auto ip_version = installed_port->value.version_string();
+                auto ap_version = available_port.version_string();
+
+                if (ip_version == ap_version) {
+                    outln("Already on latest version");
+                } else {
+                    outln("Yes -> {}", ap_version);
```

Comment:
This "Yes" is a bit out-of-context, something like "Update to {} available" would probably fit in nicer.